package.json

```
@@ -14,6 +14,7 @@
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.1.0",
     "@emotion/is-prop-valid": "^0.8.2",
+    "@emotion/styled-base": "^10.0.28",
```

```suggestion
    "@emotion/styled-base": "^10.0.27",
```

If you do this, `@emotion/styled-base` will only be included in the whole dependency tree once instead of multiple times

---

I thought about this too, I had trouble testing so I held off mentioning it. It's worth looking at.

---

You couldn't just accept the GH suggestion though, `yarn install` would have to be run again. This will basically clear up the all changes to the `yarn.lock` file.